### PR TITLE
chore(typos): fix typos

### PR DESCRIPTION
**Pull request by stavares843**
## Description of change

Fix typos across codebase.

### CODE REVIEW

File: resources/persist/src/lib.rs [hunk 1]
```diff
@@ -31,14 +31,14 @@ pub struct PersistInstance {
 }
 
 impl PersistInstance {
-    pub fn save<T: Serialize>(&self, key: &str, struc: T) -> Result<(), PersistError> {
```
Comment (oddgrd) [inline resources/persist/src/lib.rs:34]: Struct is a reserved keyword, so we can't use it here, see CI failure.

**Review by oddgrd — COMMENTED** [review]
Thanks! :partying_face: We'll just need to make a small correction for persist, I left a comment.

File: resources/persist/src/lib.rs [hunk 2]
```diff
@@ -31,14 +31,14 @@ pub struct PersistInstance {
...
+    pub fn save<T: Serialize>(&self, key: &str, struct: T) -> Result<(), PersistError> {
```
Comment (stavares843) [inline resources/persist/src/lib.rs:34]:
```suggestion
    pub fn save<T: Serialize>(&self, key: &str, struc: T) -> Result<(), PersistError> {
```

File: service/src/persist.rs [hunk 3]
```diff
@@ -30,14 +30,14 @@ pub struct PersistInstance {
 }
 
 impl PersistInstance {
-    pub fn save<T: Serialize>(&self, key: &str, struc: T) -> Result<(), PersistError> {
+    pub fn save<T: Serialize>(&self, key: &str, struct: T) -> Result<(), PersistError> {
```
Comment (stavares843) [inline service/src/persist.rs:33]:
```suggestion
    pub fn save<T: Serialize>(&self, key: &str, struc: T) -> Result<(), PersistError> {
```

File: service/src/persist.rs [hunk 4]
```diff
@@ -30,14 +30,14 @@ pub struct PersistInstance {
...
         let storage_folder = self.get_storage_folder();
         fs::create_dir_all(storage_folder).map_err(PersistError::CreateFolder)?;
 
         let file_path = self.get_storage_file(key);
         let file = File::create(file_path).map_err(PersistError::Open)?;
         let mut writer = BufWriter::new(file);
-        Ok(serialize_into(&mut writer, &struc).map_err(PersistError::Serialize))?
+        Ok(serialize_into(&mut writer, &struct).map_err(PersistError::Serialize))?
```
Comment (stavares843) [inline service/src/persist.rs:40]:
```suggestion
        Ok(serialize_into(&mut writer, &struc).map_err(PersistError::Serialize))?
```

File: resources/persist/src/lib.rs (hunk 1 above)
Comment (stavares843) [reply resources/persist/src/lib.rs]: addressed 🔨

**Review by oddgrd — APPROVED** [review]
LGTM, thanks!!

**Pull request merged**